scss/components/side-nav.scss

```diff
@@ -54,6 +54,8 @@ $block: #{$fd-namespace}-side-nav;
         text-transform: uppercase;
         color: $fd-side-nav-link-disabled-color;
         background: $fd-side-nav-title-background-color;
+        font-size: $fd-font-size;
+        font-family: $fd-font-family;
```

Comment:
give this 2 lines code because @import "../scss/fonts/72"; didn't change the font style that shown in the playground

Reply:
This component should just include the mixin `fd-reset`, which will do this and a few other things that we can no longer depend on from the top-level DOM nodes.